doc/user/content/sql/create-source/load-generator.md

```diff
@@ -110,6 +111,63 @@ The organizations, users, and accounts are fixed at the time the source
+
+    Field             | Type                         | Description
+    ------------------|------------------------------|------------
+    `id`              | [`bigint`]                   | A unique identifier for the impression.
+    `customer_id`     | [`bigint`]                   | The identifier of the customer that saw the ad. References `customers.id`.
+    `impression_time` | [`timestamp with time zone`] | The time at which the ad was seen.
+
+  * `clicks` describes clicks of ads.
+
+    Field             | Type                         | Description
+    ------------------|------------------------------|------------
+    `impression_id`   | [`bigint`]                   | The identifier of the impression that was clicked. References `impressions.id`.
+    `click_time`      | [`timestamp with time zone`] | The time at which the impression was clicked.
+
+  * `leads` describes a potential lead for a purchase.
+
+    Field               | Type                         | Description
+    --------------------|------------------------------|------------
+    `id`                | [`bigint`]                   | A unique identifier for the lead.
+    `customer_id`       | [`bigint`]                   | The identifier of the customer we'd like to convert. References `customers.id`.
+    `created_at`        | [`timestamp with time zone`] | The time at which the lead was created.
+    `converted_at`      | [`timestamp with time zone`] | The time at which the lead was converted.
+    `conversion_amount` | [`bigint`]                   | The amount the lead converted for in pennies.
+
+  * `coupons` describes coupons given to leads.
+
+    Field               | Type                         | Description
+    --------------------|------------------------------|------------
+    `id`                | [`bigint`]                   | A unique identifier for the coupon.
+    `lead_id`           | [`bigint`]                   | The identifier of the lead we're attempting to convert. References `leads.id`.
+    `created_at`        | [`timestamp with time zone`] | The time at which the coupon was created.
+    `amount`            | [`bigint`]                   | The amount the coupon is for in pennies.
+
+  * `conversion_predictions` describes the predictions made by a highly sophisticated machine learning model.
+
+    Field               | Type                         | Description
+    --------------------|------------------------------|------------
+    `lead_id`           | [`bigint`]                   | The identifier of the lead we're attempting to convert. References `leads.id`.
+    `experiment_bucket`| [`text`]                     | Whether the lead is a control or experiment.
+    `created_at`        | [`timestamp with time zone`] | The time at which the prediction was made.
+    `score`             | [`numeric`]                  | The predicted likelihood the lead will convert.
```

Comment:
Not yet sure why, but this isn't rendering as a link:
<img width="773" alt="Screenshot 2023-05-22 at 4 37 53 PM" src="https://github.com/MaterializeInc/materialize/assets/15893490/f2dea998-b4c6-4055-acaf-b5792d809689">

Reply:
Ah, I see why. Please edit `content > sql > types > numeric.md` and add this alias:

```
- /sql/types/numeric
```

Or you can change this to `[decimal]` and it should just work.

Reply:
I'm gonna go the easy route and rename it to decimal 😅

Reply:
Hum, neither will work. The fix is to add either type to the very bottom of the file, where some anchor links are defined to avoid repeating them all over.